docusaurus/docs/guides/deployments/10-linux/20-router/10-deploy.mdx

```diff
@@ -0,0 +1,112 @@
+
+### Bootstrapping the Configuration
+
+The router's configuration is loaded from a YAML file ([reference](/reference/30-configuration/router.md)). The Linux system service will generate a valid configuration during the first startup, unless one already exists. You can modify the generated configuration in `/var/lib/ziti-router/config.yml`.
+
+The filename of the configuration file, relative to the router's working directory, is given as an argument, e.g., `entrypoint.bash run config.yml` in `/lib/systemd/system/ziti-router.service` as an argument to the `ExecStart` command. You may override this in `/etc/systemd/system/ziti-router.service.d/override.conf`.
+
+Explore configuration option variables by running `ziti create config environment --help`. Any of these may be set in `/opt/openziti/etc/router/bootstrap.env` to influence the generated configuration.
+
+Disable generating a configuration by setting `ZITI_BOOTSTRAP_CONFIG=false` in `/opt/openziti/etc/router/service.env`.
+
+### Bootstrapping the Enrollment
+
+You must provide an enrollment token. Obtain the token by administratively creating the router.
+
+```text title="Create a router"
+ziti edge create edge-router "AcmeRouter1" \
+    --tunneler-enabled \
+    --jwt-output-file path/to/token.jwt
+```
+
+[Learn more about managing routers with the CLI](/guides/deployments/10-linux/20-router/40-cli-mgmt.md).
+
+The systemd service looks for the token in `/opt/openziti/etc/router/.token` if env var `ZITI_ENROLL_TOKEN` is empty. The file must be readable by root (not others).
+
+`ZITI_ENROLL_TOKEN` may be defined in `/opt/openziti/etc/router/bootstrap.env`.
+
+The router will enroll with the controller during the first startup. The one-time enrollment token is consumed during the enrollment process and a private key is generated in the router's working directory.
+
+Disable bootstrapping enrollment by setting `ZITI_BOOTSTRAP_ENROLLMENT=false` in `/opt/openziti/etc/router/service.env`.
+
+## Firewall
+
+The router listens on a single configurable TCP port: `3022/tcp`. Ziti clients use ALPN to negotiate a connection to the edge (`ziti-edge`), fabric links (`ziti-link`), or health-check APIs (`h2`, `http/1.1`).
+
+You may set `ZITI_ROUTER_PORT` in `/opt/openziti/etc/router/bootstrap.env` to bootstrap with a different port.
+
+Clients "learn" the router's address and port when they poll the controller for authorized routers, so it is not necessary to re-enroll or re-create the client if the router's address or port changes. Simply change the config, bounce the router, and it will begin advertising the new address and port. This works because the system service and Docker container both auto-renew their server certificate every startup, and the certificate's DNS subject alternative name is set in the router's config.yml file. If you set `ZITI_AUTO_RENEW_CERTS=false` then it's necessary to administratively re-create the router or run at least once with the `--extend` flag.
+
+## Agent
```

Comment:
Reference information and unnecessary in the 'install' portion. It's plausible that there's a whole troubleshooting sub page though that not only talks about HOW to use the agent (the fact that there's an 'agent' and uses IPC is reference information and not relevant to installation/management to me but maybe it's relevant insofar as permissions?)

It should cover collecting logs, acquiring a stack dump and heap dump and other such 'operational' tpye things

Reply:
I do see an opportunity to split this info between the part that's pertinent to the type of deployment (Linux, using `nsenter`) and the part that's generic to all types of deployments (the agent commands you can run for ops tasks, not just troubleshooting).

Reply:
https://github.com/openziti/ziti-doc/pull/848